refl1d/reflectivity.py

```diff
@@ -452,63 +772,67 @@ def convolve_sampled(xi, yi, xp, yp, x, dx):
     return y
 
 def test_uniform():
-    xi = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-    yi = [1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
-    _check_uniform("uniform aligned", xi, yi, x=[2, 4, 6, 8], dx=1)
-    _check_uniform("uniform unaligned", xi, yi, x=[2.5, 4.5, 6.5, 8.5], dx=1)
-    _check_uniform("uniform wide", xi, yi, x=[2.5, 4.5, 6.5, 8.5], dx=3)
+    xi=[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+    yi=[1, 3, 1, 2, 7, 3, 1, 2, 1, 3]
+    _check_uniform("uniform aligned", xi, yi, x = [2, 4, 6, 8], dx = 1)
+    _check_uniform("uniform unaligned", xi, yi,
+                   x = [2.5, 4.5, 6.5, 8.5], dx = 1)
+    _check_uniform("uniform wide", xi, yi, x = [2.5, 4.5, 6.5, 8.5], dx = 3)
     # Check bad values
-    ystar = convolve(xi, yi, x=[-3, 13], dx=[1/np.sqrt(3)]*2, resolution='uniform')
+    ystar = convolve(xi, yi, x = [-3, 13], dx = [1/np.sqrt(3)]*2, resolution ='uniform')
```

Comment:
PEP8: Don't use spaces around the = sign when used to indicate a keyword argument.

Reply:
I reformatted the whole thing with VS code autoformat.